01_fib/joel00wood/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func fibCalc(n float64) float64 {
+	switch n {
+	case 0:
+		return n
+	case 1:
+		return n
+	default:
+		return fibCalc(n-1) + fibCalc(n-2)
+	}
+}
+
+func fibN(n float64) float64 {
+	if n < 0 {
+		var i float64
+		n := math.Abs(n)
+		i = math.Pow(-1, n+1)
```

Comment:
this is definitely the most creative way I've seen to achieve this.

can you think of a less opaque way?  say, perhaps using bitwise operations?